Resources/Prototypes/Recipes/Construction/Graphs/weapons/bola.yml

```diff
@@ -0,0 +1,20 @@
+- type: constructionGraph
+  id: bola
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: bola
+          steps:
+            - tag: Handcuffs
+              icon:
+                sprite: Objects/Misc/cablecuffs.rsi
+                state: cuff
+                color: red
+              name: cuffs
+            - material: Steel
+              amount: 6
+              doAfter: 2
```

Comment:
IMO this might be a little too easy to make for a ranged stun weapon, though I'm not sure what would be better--maybe require 'actual' handcuffs + cable + steel

Reply:
ehh actually it might be fine